src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -280,9 +280,7 @@ export class TerminalService implements ITerminalService {
 				this._connectionState = TerminalConnectionState.Connected;
 			}
 
```

Comment:
this was only used by the `terminalView` before, so is safe to remove in favor of calling `_initializeTerminals` within the `terminalService` itself